source/reference/minio-mc-admin/mc-admin-cluster-iam.rst

```diff
@@ -0,0 +1,56 @@
+.. _minio-mc-admin-cluster-iam:
+
+========================
+``mc admin cluster iam``
+========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin cluster iam
+
+Description
+-----------
+
+.. versionadded:: RELEASE.2022-06-26T18-51-48Z
+
+.. start-mc-admin-cluster-iam-desc
+
+The :mc:`mc admin cluster iam` command and its subcommands provide tools for manually importing and exporting MinIO :ref:`IAM <minio-authentication-and-identity-management>` metadata.
```

Comment:
Might be worth defining `IAM` here as identity and access management.
Since it's the first time on the page.

Reply:
```suggestion
The :mc:`mc admin cluster iam` command and its subcommands provide tools for manually importing and exporting MinIO :ref:`identity and access management (IAM) <minio-authentication-and-identity-management>` metadata.
```